### PR TITLE
Add resource

### PR DESCRIPTION
**Pull request by hdadhich01**
Added a resource that uses the guide amongst other tools to set up a complete development environment.

### CODE REVIEW

File: content/articles/99999980-upgrade-python-on-linux.md [hunk 1]
```diff
@@ -156,11 +156,19 @@ sudo apt install python3.10-venv
...
 If you have [oh-my-zsh](https://ohmyz.sh/) installed, you can avoid typing out `python3` by running
 ```bash
 echo "alias py=/usr/bin/python3" >> ~/.zshrc
 echo "alias python=/usr/bin/python3" >> ~/.zshrc
 ```
-Now you can run your files with `py` or `python`.
-To help you skip the hassle in virtual environment setup and package management, consider using [PDM](https://pdm.fming.dev/latest/).
+Now you can run your files with `py` or `python`. To help you skip the hassle in virtual environment setup and package management, consider using [PDM](https://pdm.fming.dev/latest/). 
+
+You can also learn how to [set up a WSL terminal](https://github.com/hdadhich01/wsl-terminal) to have a full-fledged Python development environment.
```
Comment (rehanhaider) [inline content/articles/99999980-upgrade-python-on-linux.md:174]: I don't think this is relevant and specific enough to the article scope and should not be included.

File: content/articles/99999980-upgrade-python-on-linux.md [hunk 2]
```diff
@@ -156,11 +156,19 @@ sudo apt install python3.10-venv
...
+
 ### Extra
 If you have [oh-my-zsh](https://ohmyz.sh/) installed, you can avoid typing out `python3` by running
 ```bash
 echo "alias py=/usr/bin/python3" >> ~/.zshrc
 echo "alias python=/usr/bin/python3" >> ~/.zshrc
 ```
-Now you can run your files with `py` or `python`.
-To help you skip the hassle in virtual environment setup and package management, consider using [PDM](https://pdm.fming.dev/latest/).
+Now you can run your files with `py` or `python`. To help you skip the hassle in virtual environment setup and package management, consider using [PDM](https://pdm.fming.dev/latest/).
```
Comment (rehanhaider) [inline content/articles/99999980-upgrade-python-on-linux.md:172]: I don't think this is relevant and specific enough to the article scope and should not be included.

**Review by rehanhaider — CHANGES_REQUESTED** [review]
Please remove non-article scope specific instructions.

**Comment by hdadhich01** [comment]
The current setup removes the ability to run `sudo add-apt-repository` because we removed and reinstalled `python3-apt`. I couldn't find a way to work around this.

**Comment by rehanhaider** [comment]
Closing due to inactivity.

**Pull request closed without merging**